src/board_filter.py

```diff
@@ -0,0 +1,78 @@
+"""
+This script is intended to be used with the related filter_finder.py. It filters out all the pixels corresponding to a single skateboard in a given controlled video of a skateboard under static conditions.
+
+TODO: make this script configurable to run from the commandline with no need to edit code, e.g. through `python board_filter -v "board.mp4" -c "wheels"`
+
+This script was created as part of a school project, and therefore license-free.
```

Comment:
we should add a license into the repo, and not need license description on all files